Store & Reload the schema metadata on disk (#956)
platformatic/platformatic: docs/reference/db/configuration.md

  }
  ```

> - **`schemalock`** (`boolean` or `object`, default: `false`) — Controls the caching of the database schema on disk;
>   if set to `true` it would store the database schema metadata inside a `schema.lock` file.

Reviewer: ```suggestion
- **`schemalock`** (`boolean` or `object`, default: `false`) — Controls the caching of the database schema on disk.
  If set to `true` the database schema metadata is stored inside a `schema.lock` file.
```
